vignettes/synapser.Rmd

```diff
@@ -254,6 +254,31 @@ annotations <- synSetAnnotations(project, annotations)
 annotations
 ```
 
+## Type Conversions
+
+The core of the Synapser library is the [python client](https://python-docs.synapse.org/). [Reticulate](https://rstudio.github.io/reticulate/) is used to translate R to Python and back.
+
+These are the documented conversions from R to  Python: [Reticulate Type Conversions](https://rstudio.github.io/reticulate/articles/calling_python.html#type-conversions)
+
+`NA` values in R do not have a direct equivalent in Python. This has been a topic of great discussion. Synapser is not trying to solve this. `NA` values in R are handled by Reticulate per the implementation defined in Reticulate.
+
+When working with R and Reticulate, NA values in types tables are converted by Reticulate depending on the data type.
+
+For numeric data types, NA values are converted to the corresponding R value `NA_real_`.
+For integer data types, NA values are converted to `NA_integer_`.
+For logical data types, NA values are converted to `NA_logical_`.
+For character data types, NA values are converted to `NA_character_`.
```

Comment:
Can you explain what this means?  It would help to have an example, for instance. Conversion of boolean vectors from python to R will cause the NA's to be TRUE?

Reply:
Added NA examples for logical, char and numeric conversions